html/blog.html

```diff
@@ -0,0 +1,106 @@
+
+            <br> Duis accumsan, enim dignissim feugiat pharetra, ex enim congue sem, eu rhoncus sem diam sit amet est. Suspendisse potenti. Suspendisse et diam velit. Nunc fermentum massa felis, quis dictum elit posuere eget. Nunc quis lectus in felis aliquet venenatis eget et lorem. Praesent sed ullamcorper velit, eu eleifend orci. Vivamus mattis placerat magna non vestibulum. In hac habitasse platea dictumst. Maecenas semper, lectus ac porttitor commodo, dui diam tempus arcu, eget cursus massa mi a metus. Ut elementum elit mauris, nec porta erat lobortis eu.
+          </p>
+
+        </section>
+      </section>
+      <section class="blog3">
+        <img src="../extra/pickle.jpg" alt="pickling" />
+        <section>
+          <h2>August 20, 2017</h2>
+          <p>
+            Integer dapibus sapien nisl, eget congue dui pulvinar eu. Mauris mattis non risus a blandit. In neque enim, elementum quis accumsan non, aliquet vel tortor. Nunc iaculis varius tellus, vitae malesuada lacus tincidunt sed. Sed consequat imperdiet libero, vel posuere mauris aliquet non. Phasellus dignissim eros ac molestie volutpat. Maecenas id quam pretium, feugiat ipsum et, tempus libero. Praesent odio risus, semper eget rutrum lobortis, ullamcorper at purus. In tristique ante ultrices massa vulputate, vel fermentum nisi pretium. Sed hendrerit sagittis interdum. Morbi sagittis tincidunt efficitur. Nam dapibus lacus ac eros ultricies, ac tristique massa aliquet. Suspendisse volutpat, magna eu imperdiet ornare, velit augue sodales libero, eu laoreet leo libero mattis eros. Ut eleifend justo neque, ac rutrum mauris mattis ut. Donec cursus libero viverra eros scelerisque pulvinar. Pellentesque nisi metus, consectetur non elit in, efficitur vehicula tellus.<br>
+
+            <br> Nulla mattis dapibus felis. In ac mauris eu mi elementum congue efficitur blandit erat. Nulla neque neque, dapibus eu tortor a, volutpat egestas urna. Pellentesque vel venenatis mauris. Pellentesque luctus, nunc non pharetra molestie, dui nunc condimentum purus, eu consectetur orci dui et libero. Nulla viverra quam nec sem interdum vehicula. Proin feugiat nisl at quam tristique auctor. Nunc laoreet venenatis magna eleifend molestie. Nulla magna augue, ultricies vel fermentum ut, posuere quis orci. Mauris ac convallis ipsum.<br>
+
+            <br> Duis accumsan, enim dignissim feugiat pharetra, ex enim congue sem, eu rhoncus sem diam sit amet est. Suspendisse potenti. Suspendisse et diam velit. Nunc fermentum massa felis, quis dictum elit posuere eget. Nunc quis lectus in felis aliquet venenatis eget et lorem. Praesent sed ullamcorper velit, eu eleifend orci. Vivamus mattis placerat magna non vestibulum. In hac habitasse platea dictumst. Maecenas semper, lectus ac porttitor commodo, dui diam tempus arcu, eget cursus massa mi a metus. Ut elementum elit mauris, nec porta erat lobortis eu.
+          </p>
+
+        </section>
+      </section>
+      <section class="blog4">
```

Comment:
Section is a good tag, but I think article might be more specific. Do you agree?